iree/compiler/Codegen/LLVMCPU/LLVMCPUTileAndVectorizeLinalgTensorOps.cpp

```diff
@@ -223,6 +223,14 @@ void LLVMCPUTileAndVectorizePass::runOnOperation() {
     });
   }
 
+  {
+    RewritePatternSet vectorContractSpecialLoweringPatterns(context);
+    populateVectorContractToAArch64InlineAsm(
+        vectorContractSpecialLoweringPatterns, context);
+    (void)applyPatternsAndFoldGreedily(
```

Comment:
These casts to void are bad. Please don't spread them. This should correctly propagate the failure (return failure, signalPassFailure, etc). Tech debt that unfortunately hasn't been cleaned up: https://github.com/google/iree/issues/4759

Reply:
Thanks for letting me know!